lib/functions/modem.js

```diff
@@ -353,24 +413,22 @@ module.exports = function (SerialPort) {
       newparts = part.split('\n')
       newparts.forEach(newpart => {
         let pduTest = /[0-9A-Fa-f]{15}/g
-        if (newpart.substr(0, 6) == '+CMTI:') { // New Message Indicatpr with SIM Card ID, After Recieving Read The Message From the SIM Card
-          newpart = newpart.split(',')
-          modem.readSMSById(newpart[1], res => { })
-        } else if (newpart.substr(0, 5) == '+CLIP') { // New Message Indicatpr with SIM Card ID, After Recieving Read The Message From the SIM Card
-          newpart = newpart.split(',')
+        if (newpart.substr(0, 6) == '+CMTI:') { // New Message Indicator with SIM Card ID, After Recieving Read The Message From the SIM Card
```

Comment:
with these setting, it does not give the error.

Reply:
@karianpour that's good!